js/app.js

```diff
@@ -100,6 +100,13 @@ $(document).ready(function(){
             'soft pretzel',
             'soda',
             'Cheez Whiz',
+            'wooder',
+            'youse',
+            'Poconos',
+            'Rocky',
+            'friggin',
+            'strut',
+            'Iggles'
```

Comment:
iggles is already up top. Arguably all of these should be lowercase, so they can be used in any context; then there should css (```:first-letter``` or something) to fix them at the beginning of sentences.